### PR TITLE
CCO05948-Knoxville

### PR DESCRIPTION
**Pull request by CCO05948**
This pull request adds a file about Knoxville, TN and where to eat and visit. 🍸

### CODE REVIEW

File: CCO05948-Knoxville.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Knoxville, TN
+
+## Places to Eat
+- Mirage of Knoxville for baba ganoush and mixed drinks
+- Pizza Palace
+- Buddy's BBQ
+
```
Comment (robertrankin) [inline CCO05948-Knoxville.md]: Looks good. BBQ is the shizzle!

**Review by robertrankin — COMMENTED** [review]
Good work

**Review by robertrankin — APPROVED** [review]
@CCO05948 approved

File: CCO05948-Knoxville.md (hunk 1 above)
Comment (CCO05948) [reply CCO05948-Knoxville.md]: @robertrankin can you look over the recent changes I made? Thank you! 😃

**Pull request merged**